ocr/google_docai_provider_test.go

```
@@ -0,0 +1,99 @@
+			}
+			if !strings.Contains(result, "<html xmlns=\"http://www.w3.org/1999/xhtml\"") {
+				t.Error("missing HTML namespace")
+			}
+			if !strings.Contains(result, "<meta name='ocr-system' content='google-docai'") {
+				t.Error("missing OCR system metadata")
+			}
+		})
+	}
+}
+
+func testContext() context.Context {
```

golangci-lint (1.62.2)

func `testContext` is unused

(unused)